.gitignore

```diff
@@ -1,3 +1,6 @@
 gopath/
 bin/
 *.swp
+*.img
```

Comment:
Yeah, put your scratch directory in `/var/tmp`.  You may want to do it only if TMPDIR isn't custom, so the user can still override.